react-client/src/components/Timer.jsx

```diff
@@ -4,14 +4,57 @@ class Timer extends React.Component {
 
   constructor(props) {
     super(props);
-    this.seconds = props.seconds
+    //this.seconds = props.seconds
+     this.state = {seconds: 30 };
```

Comment:
This is not the right thing. Some timers in the interface will be 30s. Others will be multiple minutes (1 minute per spy is what we discussed). In the various places a `<Timer>` component is invoked, I was passing down the appropriate number of seconds.